.env.sample

```diff
@@ -0,0 +1,2 @@
+GITHUB_TOKEN=
```

Comment:
Add comments or documentation explaining the purpose of these environment variables and how they should be set or configured, to enhance maintainability and help other developers understand their use.